.github/workflows/ci.yml

```diff
@@ -44,44 +43,24 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: [3.7, 3.8]
+        os: [ubuntu-latest, windows-latest, macos-latest, macos-11.0]
+        python-version: [3.7, 3.8, 3.9, '3.10-dev']
         browser: [chromium, firefox, webkit]
-        include:
-        - os: ubuntu-latest
-          python-version: 3.9
-          browser: chromium
-        - os: windows-latest
-          python-version: 3.9
```

Comment:
The reason for manually including it previously was that we ended up having too many jobs. 58 is quite a lot...

Reply:
Now it would be even more :)